### PR TITLE
improve LT category mapping, specifically map LT style to style category

### LINKED ISSUE #596: review LT rules
I just noticed that LanguageTool picks up on `super Signal` as `COLLOQUIALISMS` \.\. should we put that into `style` , subcategory `colloquialisms`? do we even think this is a problem? it currently is shown as a spelling

<p></p>

in generally we might have to spend more time looking at LT rules, deciding which to keep enabled and how we best map them to our \(sub\)\-categories

.png)

<https://languagetool.org/development/api/org/languagetool/rules/Categories.html>

disable:
* COLLOQUIALISMS
* GENDER_NEUTRALITY 

map to style:
* PLAIN_ENGLISH
* FALSE_FRIENDS
* REDUNDANCY
* REGIONALISMS
* REPETITIONS_STYLE 
* SEMANTICS (?)
* STYLE

### PR DESCRIPTION
**Pull request by lsmith77**
see also https://wittyworks.slack.com/archives/C010GGET27M/p1664213315314329

fixes https://github.com/witty-works/nlp_api/issues/596

### CODE REVIEW

File: tests/test_lemmatizers/test_english_lemmatizer/output.json [hunk 1]
```diff
@@ -40,64 +40,14 @@
...
-            "category": "style",
-            "context": "Our efforts continue to be geared to offering the best state-of-the-art technology",
-            "end": 71,
-            "explanation": {
-                "icon": "🙄",
-                "text": "Empty words have lost all meaning",
-                "url": "https://www.witty.works/en/categories/style#empty_words"
-            },
-            "gravity": 3.0,
-            "label": "Style: Empty words",
```
Comment (lnazarenko) [inline tests/test_lemmatizers/test_english_lemmatizer/output.json:97]: @lsmith77 Why "state-of-the-art" is not highlighted here?

File: tests/test_sentry_examples/test_english_morph_token/output.json [hunk 2]
```diff
@@ -117,40 +117,14 @@
...
+                "text": "People are more likely to picture men here",
                 "url": "https://www.witty.works/en/categories/gendered#special_functions"
             },
             "gravity": 2.0,
             "label": "Gendered: Special functions",
             "start": 57,
             "subcategory": "function",
             "text": "intern"
-        },
-        {
```
Comment (lsmith77) [inline tests/test_sentry_examples/test_english_morph_token/output.json:129]: filtered out due to importance 3.0

File: tests/test_lemmatizers/test_english_lemmatizer/output.json [hunk 3]
```diff
@@ -40,64 +40,14 @@
...
+                "text": "Build trust by avoiding exaggerations",
                 "url": "https://www.witty.works/en/categories/style#exaggeration"
             },
             "gravity": 2.0,
             "label": "Style: Exaggeration",
             "start": 50,
             "subcategory": "exaggerating",
             "text": "best"
-        },
-        {
```
Comment (lsmith77) [inline tests/test_lemmatizers/test_english_lemmatizer/output.json:52]: filtered out due to importance 3.0

File: tests/test_general_cases/test_api_text_max_length/output.json [hunk 4]
```diff
@@ -413,46 +413,6 @@
             "subcategory": "communal",
             "text": "team"
         },
-        {
-            "alternatives": [
```
Comment (lsmith77) [inline tests/test_general_cases/test_api_text_max_length/output.json:417]: filtered out due to importance 3.0

**Review by lnazarenko — APPROVED** [review]

File: tests/test_lemmatizers/test_english_lemmatizer/output.json (hunk 1 above)
Comment (lsmith77) [reply tests/test_lemmatizers/test_english_lemmatizer/output.json:97]: filtered out due to importance 3.0

**Pull request merged**